tests/unit/button/button_options.js

```diff
@@ -25,135 +25,80 @@ test( "disabled, explicit value", function() {
-		} else {
-			ok( !buttonElement.hasClass( "ui-state-active" ),
-				"Disabled [" + elementType + "] does not have ui-state-active class." );
-		}
-	});
+	strictEqual( $("#button").prop("disabled"), false, "element is disabled");
 });
 
-test("text false without icon", function() {
+test( "showLabel false without icon", function() {
 	expect( 1 );
-	$("#button").button({
-		text: false
+	$( "#button" ).button({
+		showLabel: false
 	});
-	ok( $("#button").is(".ui-button-text-only:not(.ui-button-icon-only)") );
+	strictEqual( $( "#button" ).is( ":ui-button.ui-corner-all.ui-widget" ), true,
+		"Button has correct classes" );
 
-	$("#button").button("destroy");
+	$( "#button" ).button( "destroy" );
 });
 
-test("text false with icon", function() {
+test("showLabel false with icon", function() {
 	expect( 1 );
 	$("#button").button({
-		text: false,
-		icons: {
-			primary: "iconclass"
-		}
+		showLabel: false,
+		icon: "iconclass"
 	});
-	ok( $("#button").is(".ui-button-icon-only:not(.ui-button-text):has(span.ui-icon.iconclass)") );
-
-	$("#button").button("destroy");
+	strictEqual( $( "#button" ).is( ".ui-button.ui-corner-all.ui-widget.ui-button-icon-only" ),
+		true, "Button has correct classes" );
+	$( "#button" ).button( "destroy" );
 });
 
-test("label, default", function() {
+test( "label, default", function() {
 	expect( 2 );
-	$("#button").button();
-	deepEqual( $("#button").text(), "Label" );
-	deepEqual( $( "#button").button( "option", "label" ), "Label" );
+	$( "#button" ).button();
+	deepEqual( $( "#button" ).text(), "Label" );
+	deepEqual( $( "#button" ).button( "option", "label" ), "Label" );
 
-	$("#button").button("destroy");
+	$( "#button" ).button( "destroy" );
 });
 
-test("label", function() {
+test( "label", function() {
 	expect( 2 );
-	$("#button").button({
+	$( "#button" ).button({
 		label: "xxx"
 	});
-	deepEqual( $("#button").text(), "xxx" );
-	deepEqual( $("#button").button( "option", "label" ), "xxx" );
+	deepEqual( $( "#button" ).text(), "xxx" );
+	deepEqual( $( "#button" ).button( "option", "label" ), "xxx" );
 
-	$("#button").button("destroy");
+	$( "#button" ).button( "destroy" );
 });
 
-test("label default with input type submit", function() {
+test( "label default with input type submit", function() {
 	expect( 2 );
-	deepEqual( $("#submit").button().val(), "Label" );
-	deepEqual( $("#submit").button( "option", "label" ), "Label" );
+	deepEqual( $( "#submit" ).button().val(), "Label" );
+	deepEqual( $( "#submit" ).button( "option", "label" ), "Label" );
 });
 
-test("label with input type submit", function() {
+test( "label with input type submit", function() {
 	expect( 2 );
-	var label = $("#submit").button({
+	var label = $( "#submit" ).button({
 		label: "xxx"
 	}).val();
 	deepEqual( label, "xxx" );
-	deepEqual( $("#submit").button( "option", "label" ), "xxx" );
+	deepEqual( $( "#submit" ).button( "option", "label" ), "xxx" );
 });
 
-test("icons", function() {
+test( "icons", function() {
```

Comment:
Let's rename this to "icon"? Or something else, since there are no tests for `iconPosition`.